src/main/java/com/project/smartStore/security/EncryptionConverter.java

```diff
@@ -23,7 +23,7 @@ public String ConvertSHA256WithSalt(String inputString) {
       byte[] bytes = messageDigest.digest(inputWithSalt.getBytes(StandardCharsets.UTF_8));
       return bytesToHex(bytes);
     } catch (NoSuchAlgorithmException e) {
-      throw new RuntimeException();
+      throw new IllegalAlgorithmException(e.getMessage(), e.getCause());
```

Comment:
이런식으로 가보면 어떨까요? 왜 이런 방식이 더 나은지도 고민해보시면 좋을 것 같네요

```suggestion
      throw new IllegalAlgorithmException("암호화에 실패하였습니다", e);
```

Reply:
예외에 대한 구체적인 메세지를 입력하여 예외 발생시 디버깅이 수월해질 것 같습니다.

Reply:
예외에서 cause란 어떤걸까요? 이걸 넘기면 예외가 발생했을 때 우리 로그에는 어떻게 보일까요?